### ISSUE TITLE
Npm run export fails

### ISSUE THREAD

**Issue by travisBradfield** [rowaxl/nextjs-with-firebase-hosting-functions#1]
Hello! 

How did you run npm export? 

I've checked out your code as is and followed your instructions in the readme. However no matter what I get the same error: 

`Error: Error for page /: pages with `getServerSideProps` can not be exported.`

Any guidance on how to avoid this?

**Comment by rowaxl** [comment]
Hi @travisBradfield !

Thanks for check my code!
First of all, **You cannot next export with getServerSideProps**.
`getServerSideProps` is used for SSR(dev, build), not SSG(export).
If you want to use SSG, please uncomment `getStaticProps` and remove `getServerSideProps`.

Have a nice day!

**Comment by travisBradfield** [comment]
@rowaxl 

You're right! I made the changes you've suggested and it works as expected. Thanks brother.
